app/containers/ViewQuestion/BountyPopover.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { BG_LIGHT, BORDER_PRIMARY_LIGHT, TEXT_DARK } from 'style-constants';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+
+const Base = styled.div`
+  position: absolute;
+  background-color: ${BG_LIGHT};
+  width: 290px;
+  text-align: left;
+  z-index: 100;
+  top: 30px;
+  border-radius: 5px;
```

Comment:
For border-radius we have some constants, because for special community style(ontology, telos ets.) it will be another value